thinkhazard/templates/report.jinja2

```diff
@@ -80,13 +80,13 @@ Think Hazard - {{ division.name}}
     <div class="row">
       <div class="col-md-7">
         {% if current_hazard %}
-          <h1 class="page-header detail level-{{ current_hazard.categorytype.mnemonic }}">
+          <h2 class="page-header detail level-{{ current_hazard.categorytype.mnemonic }}">
```

Comment:
We should remove the ".level-XXX" class here since it's already defined at the container level.